## HYPOTHESIS TESTING

##### Background:
Bombay hospitality Ltd. operates a franchise model for producing exotic Norwegian dinners throughout New England. The operating cost for a franchise in a week (W) is given by the equation W = $1,000 + $5X, where X represents the number of units produced in a week. Recent feedback from restaurant owners suggests that this cost model may no longer be accurate, as their observed weekly operating costs are higher.

##### Objective:
To investigate the restaurant owners' claim about the increase in weekly operating costs using hypothesis testing.

##### Data Provided:
•	The theoretical weekly operating cost model: W = $1,000 + $5X

•	Sample of 25 restaurants with a mean weekly cost of Rs. 3,050

•	Number of units produced in a week (X) follows a normal distribution with a mean (μ) of 600 units and a standard deviation (σ) of 25 units


In [1]:
#import math and scipy libraries
import math
from scipy import stats

### 1. State the Hypotheses statement:

##### Null Hypothesis (H0): The weekly operating cost follows the theoretical cost 
pop_mean=1000+5𝑋=1000+5*600=4000

##### Alternative Hypothesis (H1): The actual weekly operating cost is higher than the theoretical cost 
pop_mean>4000

### 2. Calculate the Test Statistic:

In [2]:
sample_mean=3050 #Sample mean
pop_mean=4000 #Population mean
std=125 #standard deviation
s_size=25 #Sample size

#Calculate t_statistic value
t_statistic=(sample_mean-pop_mean)/(std/math.sqrt(s_size))
print("t_statistic value:",t_statistic)

t_statistic value: -38.0


### 3. Determine the Critical Value:

In [3]:
#alpha value
alpha = 0.05

#Calculate t_critical value
t_critical = stats.norm.ppf(1-alpha)
print("t_critical value:",t_critical)

t_critical value: 1.6448536269514722


### 4. Make a Decision:

In [5]:
# Decision
if t_statistic > t_critical:
    print(f"Dicision - Reject the null hypothesis: {t_statistic} > {t_critical:.3f}")
else:
    print(f"Dicision - Fail to reject the null hypothesis: {t_statistic} < {t_critical:.3f}")

Dicision - Fail to reject the null hypothesis: -38.0 < 1.645


### 5. Conclusion:

##### Final Conclusion
- There is no strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.
- Hence ,It Failed to reject the null hypothesis:  (t_statistic < t_critical = -38.0 < 1.645)